config/base.js

```diff
@@ -59,6 +60,6 @@ module.exports = {
     'space-infix-ops': 'error',
     'template-curly-spacing': 'error',
 
-    'no-only-tests/no-only-tests': 2
+    'qunit/require-expect': 'off'
```

Comment:
Let's disable [no-op-equality](https://github.com/platinumazure/eslint-plugin-qunit/blob/master/docs/rules/no-ok-equality.md) also. I do that frequently when comparing Ember objects since the diff is so massive and worthless, and sometimes causes the test to fail because it has some recursive structure that blows out the stack.